### PR TITLE
Teacher can give multiple unlock conditions

### PR DESCRIPTION
**Pull request by Termanty**
Simpler way to create multiple unlock condition from UI.

### CODE REVIEW

File: app/views/courses/manage_unlocks.erb [hunk 1]
```diff
@@ -1,43 +1,69 @@
-<h1>Edit unlock dates</h1>
```
Comment (jsopakar) [inline app/views/courses/manage_unlocks.erb]: Filename missing html part?

Comment (jruo) [inline app/views/courses/manage_unlocks.erb]: Fixed

File: app/controllers/courses_controller.rb [hunk 2]
```diff
@@ -180,7 +180,7 @@ def save_unlocks
     groups = group_params
     groups.each do |name, conditions|
       array = []
-      conditions.each { |k, v| array << v }
+      conditions.each { |k, v| array << v unless v.blank? }
```
Comment (jamox) [inline app/controllers/courses_controller.rb]: I'm wondering if we have better way for doing this; `.flatten!` and `.uniq!` might be helpful.

**Comment by jruo** [comment]
Updated

**Comment by jruo** [comment]
Added button to remove conditions (https://github.com/TMC-teacher-tools/tmc-server/issues/95)

**Comment by jamox** [comment]
Merged manually to current master.

**Pull request closed without merging**